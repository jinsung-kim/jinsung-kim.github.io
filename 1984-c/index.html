<!DOCTYPE html>
<html>
  <head>
    <title>1984 (Classics Week) – :) – Jin reviews books</title>

        <meta charset="utf-8" />
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>

    
    <meta name="description" content="
  Power is in tearing human minds to pieces and putting them together again in new shapes of your own choosing.

" />
    <meta property="og:description" content="
  Power is in tearing human minds to pieces and putting them together again in new shapes of your own choosing.

" />
    
    <meta name="author" content=":)" />

    
    <meta property="og:title" content="1984 (Classics Week)" />
    <meta property="twitter:title" content="1984 (Classics Week)" />
    

    <!--[if lt IE 9]>
      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <link rel="stylesheet" type="text/css" href="/style.css" />
    <link rel="alternate" type="application/rss+xml" title=":) - Jin reviews books" href="/feed.xml" />

    <!-- Created with Jekyll Now - http://github.com/barryclark/jekyll-now -->
  </head>

  <body>
    <div class="wrapper-masthead">
      <div class="container">
        <header class="masthead clearfix">
          <a href="/" class="site-avatar"><img src="https://jsk-blog-images.s3.us-west-2.amazonaws.com/books.jpg" /></a>

          <div class="site-info">
            <h1 class="site-name"><a href="/">:)</a></h1>
            <p class="site-description">Jin reviews books</p>
          </div>

          <nav>
            <a href="/">Home</a>
            <a href="/reviews">Index</a>
            <a href="/about">About</a>
          </nav>
        </header>
      </div>
    </div>

    <div id="main" role="main" class="container">
      <article class="post">
  <h1>1984 (Classics Week)</h1>

  <div class="entry">
    <blockquote>
  <p>Power is in tearing human minds to pieces and putting them together again in new shapes of your own choosing.</p>
</blockquote>

<blockquote>
  <p>War is peace. Slavery is freedom. Ignorance is strength.</p>
</blockquote>

<p><img src="https://upload.wikimedia.org/wikipedia/commons/6/6b/1984-Big-Brother.jpg" alt="Big Brother" /></p>

<p>For the third day of Classics Week, I am reposting my review for 1984. I find it to be more relevant with Apple’s guideline changes and some of the leaks that have been going on in recent news.</p>

<p>1984 is one of the most influential and well known books written in the 20th century written by George Orwell. It was conceived in a time of intense social and political unrest, which remains relevant to this day. The novel follows Winston Smith, a low level member of the nation of Oceania’s ruling party.</p>

<p>The setting of this book is what is most important. Everywhere Winston goes, he is observed through telescreens and microphones. Even his criminal thoughts (referred to as thoughtcrimes), can be interpreted and used against him. This overseeing body is referred to as Big Brother, and anyone that steps out of line, is caught and thrown into a rehabilitation system.</p>

<p>Winston’s job is to change all documents and media to match Big Brother’s agenda. It doesn’t matter how ridiculous he rewrites history, as everyone is brainwashed to believe that Big Brother is correct. His colleague has an even crazier task - to reconstruct the language that the people speak into Newspeak - by creating a new official dictionary that renders it impossible to think illegal thoughts.</p>

<p>I never thought I would like a dystopian novel, but I have to say I thoroughly enjoyed it. I really only had two complaints with this book. First off, the characters are pretty dry. The characters lack depth, and often felt a bit hard to connect with. Additionally, it certainly felt like the rhetoric was quite didactic.</p>

<p>While I thought this book was way too grim and unrealistic at first, after reading around 100 pages, I began to see some parallels to our world. Our governments can track everything that we do, we all own smartphones that contain unique hashes and IP addresses that trace everywhere we are. Advertisers know us better than we know ourselves, and we are brainwashed by deep learning algorithms meant to keep us in a constant loop of engagement. Are we really that far off from the basic premise of Big Brother?</p>

<p>This book is a social commentary, a cautionary tale to the readers, to think twice about giving any entity too much power over our lives. I left this book with a lot more reflecting thoughts and questions. I am genuinely curious - how did Orwell predict more than half a century of his future?</p>

  </div>

  <div class="date">
    Written on October 12, 2021
  </div>

  
</article>

    </div>

    <div class="wrapper-footer">
      <div class="container">
        <footer class="footer">
          
<a href="mailto:jinsungkim1210@gmail.com"><i class="svg-icon email"></i></a>


<a href="https://github.com/jinsung-kim"><i class="svg-icon github"></i></a>
<a href="https://instagram.com/codejin_"><i class="svg-icon instagram"></i></a>







        </footer>
      </div>
    </div>

    

    <script>
    !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.crossOrigin="anonymous",p.async=!0,p.src=s.api_host.replace(".i.posthog.com","-assets.i.posthog.com")+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="init capture register register_once register_for_session unregister unregister_for_session getFeatureFlag getFeatureFlagPayload isFeatureEnabled reloadFeatureFlags updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures on onFeatureFlags onSurveysLoaded onSessionId getSurveys getActiveMatchingSurveys renderSurvey canRenderSurvey canRenderSurveyAsync identify setPersonProperties group resetGroups setPersonPropertiesForFlags resetPersonPropertiesForFlags setGroupPropertiesForFlags resetGroupPropertiesForFlags reset get_distinct_id getGroups get_session_id get_session_replay_url alias set_config startSessionRecording stopSessionRecording sessionRecordingStarted captureException loadToolbar get_property getSessionProperty createPersonProfile opt_in_capturing opt_out_capturing has_opted_in_capturing has_opted_out_capturing clear_opt_in_out_capturing debug getPageViewId captureTraceFeedback captureTraceMetric".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
    posthog.init('phc_Uma7UMxUPzkDbfuAvRdFGKmaUeKv3orygcVWlTHokDl', {
        api_host: 'https://us.i.posthog.com',
        person_profiles: 'identified_only', // or 'always' to create profiles for anonymous users as well
    })
</script>

  </body>
</html>
