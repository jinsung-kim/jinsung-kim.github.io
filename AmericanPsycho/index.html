<!DOCTYPE html>
<html>
  <head>
    <title>American Psycho – :) – Jin reviews books</title>

        <meta charset="utf-8" />
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>

    
    <meta name="description" content="
  I’m on the verge of tears by the time we arrive at Espace, since I’m positive we won’t have a decent table. But we do, and relief washes over me in an awesome wave.

" />
    <meta property="og:description" content="
  I’m on the verge of tears by the time we arrive at Espace, since I’m positive we won’t have a decent table. But we do, and relief washes over me in an awesome wave.

" />
    
    <meta name="author" content=":)" />

    
    <meta property="og:title" content="American Psycho" />
    <meta property="twitter:title" content="American Psycho" />
    

    <!--[if lt IE 9]>
      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <link rel="stylesheet" type="text/css" href="/style.css" />
    <link rel="alternate" type="application/rss+xml" title=":) - Jin reviews books" href="/feed.xml" />

    <!-- Created with Jekyll Now - http://github.com/barryclark/jekyll-now -->
  </head>

  <body>
    <div class="wrapper-masthead">
      <div class="container">
        <header class="masthead clearfix">
          <a href="/" class="site-avatar"><img src="https://jsk-blog-images.s3.us-west-2.amazonaws.com/books.jpg" /></a>

          <div class="site-info">
            <h1 class="site-name"><a href="/">:)</a></h1>
            <p class="site-description">Jin reviews books</p>
          </div>

          <nav>
            <a href="/">Home</a>
            <a href="/reviews">Index</a>
            <a href="/about">About</a>
          </nav>
        </header>
      </div>
    </div>

    <div id="main" role="main" class="container">
      <article class="post">
  <h1>American Psycho</h1>

  <div class="entry">
    <blockquote>
  <p>I’m on the verge of tears by the time we arrive at Espace, since I’m positive we won’t have a decent table. But we do, and relief washes over me in an awesome wave.</p>
</blockquote>

<p><img src="https://m.media-amazon.com/images/M/MV5BMjE2NzIwMjU4OF5BMl5BanBnXkFtZTgwMDI0NzU5MTE@._V1_.jpg" alt="Patrick Bateman" /></p>

<p>American Psycho is a book about Patrick Bateman, a vice president at Pierce &amp; Pierce on Wall Street. He is a handsome, sophisticated, and charming man, but there is something that others don’t know about him - that he is a brutal serial killer. It is novel written by Bret Easton Ellis, and was notoriously taken down from his original publisher for its violence.</p>

<p>I would never recommend this book to anyone. It has a weird flow, full of long meaningless dialogues and has a stream of consciousness narration style that makes it a difficult read at times. It’s also filled with the most horrible schemes and murders that made my stomach turn. As far as plot, there is very little - after the first 100 pages of the book, there is little point in reading on, as it just seems to be the same thing over and over again (with increasing violence and madness).</p>

<p>So, what makes this book interesting? Patrick is a textbook example of a psychopath, and his thought process that is revealed through the narration, is immensely interesting to dissect. His reactions to small things, his inner dialogue of how he wants to kill people, and the way he treats others, all seem to point to a deeper problem within him. Through the book, I felt that his actions all had deeper meanings and intents.</p>

<p>One thing I also really enjoyed was that the narration was extremely inconsistent. The author himself stated that he was not sure if Patrick really committed the murders, or simply thought them out in his mind. There seems to be no consequences for the murders that he acts out, which may hint that they were never actually committed. There are bits of clues that Patrick leaves behind, but in the end, the author really leaves it up to the reader to decide.</p>

<p>Bret Easton Ellis also has a brilliant way of pointing out the flaws of this society and time period. There are many hilarious and satirical moments between the characters, and at a certain point, you start to feel bad for the bankers, because their lives are completely meaningless. Their happiness is what brand their suit is, where they are able to get dinner reservations, and how much their annual bonus comes out to be. They are defined by the jewels that they wear and the yachts that they party on. Class is essentially a competition, a sport-like system, reflective of Thorstein Veblen’s theory of the “leisure class”. Ellis criticizes mass consumerism in the United States in a very elegant way, and points out how damaging a lack of individuality can be to society overall.</p>

<p>In conclusion, I never want to read this again, but I have to admit it was pretty interesting read. I ended up reading the whole book in just two days, not because it was particularly great, but because each page left me in a state of suspense. I might check out the movie because it stars Christian Bale.</p>

<p><strong>Overall Rating: 7/10</strong></p>

  </div>

  <div class="date">
    Written on August 14, 2021
  </div>

  
</article>

    </div>

    <div class="wrapper-footer">
      <div class="container">
        <footer class="footer">
          
<a href="mailto:jinsungkim1210@gmail.com"><i class="svg-icon email"></i></a>


<a href="https://github.com/jinsung-kim"><i class="svg-icon github"></i></a>
<a href="https://instagram.com/codejin_"><i class="svg-icon instagram"></i></a>







        </footer>
      </div>
    </div>

    

    <script>
    !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.crossOrigin="anonymous",p.async=!0,p.src=s.api_host.replace(".i.posthog.com","-assets.i.posthog.com")+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="init capture register register_once register_for_session unregister unregister_for_session getFeatureFlag getFeatureFlagPayload isFeatureEnabled reloadFeatureFlags updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures on onFeatureFlags onSurveysLoaded onSessionId getSurveys getActiveMatchingSurveys renderSurvey canRenderSurvey canRenderSurveyAsync identify setPersonProperties group resetGroups setPersonPropertiesForFlags resetPersonPropertiesForFlags setGroupPropertiesForFlags resetGroupPropertiesForFlags reset get_distinct_id getGroups get_session_id get_session_replay_url alias set_config startSessionRecording stopSessionRecording sessionRecordingStarted captureException loadToolbar get_property getSessionProperty createPersonProfile opt_in_capturing opt_out_capturing has_opted_in_capturing has_opted_out_capturing clear_opt_in_out_capturing debug getPageViewId captureTraceFeedback captureTraceMetric".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
    posthog.init('phc_Uma7UMxUPzkDbfuAvRdFGKmaUeKv3orygcVWlTHokDl', {
        api_host: 'https://us.i.posthog.com',
        person_profiles: 'identified_only', // or 'always' to create profiles for anonymous users as well
    })
</script>

  </body>
</html>
