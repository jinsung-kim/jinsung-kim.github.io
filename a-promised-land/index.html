<!DOCTYPE html>
<html>
  <head>
    <title>A Promised Land – :) – Jin reviews books</title>

        <meta charset="utf-8" />
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>

    
    <meta name="description" content="
  I suspect that God’s plan, whatever it is, works on a scale too large to admit our mortal tribulations; that in a single lifetime, accidents and happenstance determine more than we care to admit; and that the best we can do is to try to align ourselves with what we feel is right and construct some meaning out of our confusion, and with grace and nerve play at each moment the hand that we’re dealt.

" />
    <meta property="og:description" content="
  I suspect that God’s plan, whatever it is, works on a scale too large to admit our mortal tribulations; that in a single lifetime, accidents and happenstance determine more than we care to admit; and that the best we can do is to try to align ourselves with what we feel is right and construct some meaning out of our confusion, and with grace and nerve play at each moment the hand that we’re dealt.

" />
    
    <meta name="author" content=":)" />

    
    <meta property="og:title" content="A Promised Land" />
    <meta property="twitter:title" content="A Promised Land" />
    

    <!--[if lt IE 9]>
      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <link rel="stylesheet" type="text/css" href="/style.css" />
    <link rel="alternate" type="application/rss+xml" title=":) - Jin reviews books" href="/feed.xml" />

    <!-- Created with Jekyll Now - http://github.com/barryclark/jekyll-now -->
  </head>

  <body>
    <div class="wrapper-masthead">
      <div class="container">
        <header class="masthead clearfix">
          <a href="/" class="site-avatar"><img src="https://jsk-blog-images.s3.us-west-2.amazonaws.com/books.jpg" /></a>

          <div class="site-info">
            <h1 class="site-name"><a href="/">:)</a></h1>
            <p class="site-description">Jin reviews books</p>
          </div>

          <nav>
            <a href="/">Home</a>
            <a href="/reviews">Index</a>
            <a href="/about">About</a>
          </nav>
        </header>
      </div>
    </div>

    <div id="main" role="main" class="container">
      <article class="post">
  <h1>A Promised Land</h1>

  <div class="entry">
    <blockquote>
  <p>I suspect that God’s plan, whatever it is, works on a scale too large to admit our mortal tribulations; that in a single lifetime, accidents and happenstance determine more than we care to admit; and that the best we can do is to try to align ourselves with what we feel is right and construct some meaning out of our confusion, and with grace and nerve play at each moment the hand that we’re dealt.</p>
</blockquote>

<p><img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fimagesvc.meredithcorp.io%2Fv3%2Fmm%2Fimage%3Furl%3Dhttps%3A%252F%252Fstatic.onecms.io%252Fwp-content%252Fuploads%252Fsites%252F6%252F2020%252F09%252F17%252FA-Promised-Land-2000.jpg&amp;f=1&amp;nofb=1" alt="Book ironically costs 45 dollars" /></p>

<p>As someone who was too young to understand Obama’s presidency, this book was a very fascinating read. A Promised Land is a memoir written by Barack Obama himself, as he tells his journey of becoming a thoughtful young man, his crazy rise into the politics, and his transition and service as the leader of the free world. He offers a very unique exploration of his presidency - the powers that he had, and the challenges that he faced throughout this election cycle and first term.</p>

<p>The book is extremely intimate and introspective, and Obama’s honesty reminded me that we are all just human at the end of the day. Obama talks about his poor decisions, and reflects upon how he should have acted differently, and even doubts his position and whether he belongs there or not. It served as a reminder that we should be realistic with what we expect from our political leaders.</p>

<p>I wonder what would have happened in Obama never got into politics, or if he had never been elected president. What would the grassroots campaign look like in Chicago? Would his successor (Donald Trump) still run? Obama hints at these possibilities and events in a very elegant way.</p>

<p>Overall, this book is written beautifully, and is worth the read. If you are curious about any of his controversies and events, such as his health care act, the economic crisis, or foreign policy, this book is definitely the right place to look. While it is certainly long, there are so many little nuggets of knowledge and experience that make it worthwhile.</p>

<p><strong>Overall Rating: 8/10</strong></p>

  </div>

  <div class="date">
    Written on August 30, 2021
  </div>

  
</article>

    </div>

    <div class="wrapper-footer">
      <div class="container">
        <footer class="footer">
          
<a href="mailto:jinsungkim1210@gmail.com"><i class="svg-icon email"></i></a>


<a href="https://github.com/jinsung-kim"><i class="svg-icon github"></i></a>
<a href="https://instagram.com/codejin_"><i class="svg-icon instagram"></i></a>







        </footer>
      </div>
    </div>

    

    <script>
    !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.crossOrigin="anonymous",p.async=!0,p.src=s.api_host.replace(".i.posthog.com","-assets.i.posthog.com")+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="init capture register register_once register_for_session unregister unregister_for_session getFeatureFlag getFeatureFlagPayload isFeatureEnabled reloadFeatureFlags updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures on onFeatureFlags onSurveysLoaded onSessionId getSurveys getActiveMatchingSurveys renderSurvey canRenderSurvey canRenderSurveyAsync identify setPersonProperties group resetGroups setPersonPropertiesForFlags resetPersonPropertiesForFlags setGroupPropertiesForFlags resetGroupPropertiesForFlags reset get_distinct_id getGroups get_session_id get_session_replay_url alias set_config startSessionRecording stopSessionRecording sessionRecordingStarted captureException loadToolbar get_property getSessionProperty createPersonProfile opt_in_capturing opt_out_capturing has_opted_in_capturing has_opted_out_capturing clear_opt_in_out_capturing debug getPageViewId captureTraceFeedback captureTraceMetric".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
    posthog.init('phc_Uma7UMxUPzkDbfuAvRdFGKmaUeKv3orygcVWlTHokDl', {
        api_host: 'https://us.i.posthog.com',
        person_profiles: 'identified_only', // or 'always' to create profiles for anonymous users as well
    })
</script>

  </body>
</html>
