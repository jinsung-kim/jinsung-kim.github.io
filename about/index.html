<!DOCTYPE html>
<html>
  <head>
    <title>About – :) – Jin reviews books</title>

        <meta charset="utf-8" />
    <meta content='text/html; charset=utf-8' http-equiv='Content-Type'>
    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0'>

    
    <meta name="description" content="Jin reviews books">
    <meta property="og:description" content="Jin reviews books" />
    
    <meta name="author" content=":)" />

    
    <meta property="og:title" content="About" />
    <meta property="twitter:title" content="About" />
    

    <!--[if lt IE 9]>
      <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->

    <link rel="stylesheet" type="text/css" href="/style.css" />
    <link rel="alternate" type="application/rss+xml" title=":) - Jin reviews books" href="/feed.xml" />

    <!-- Created with Jekyll Now - http://github.com/barryclark/jekyll-now -->
  </head>

  <body>
    <div class="wrapper-masthead">
      <div class="container">
        <header class="masthead clearfix">
          <a href="/" class="site-avatar"><img src="https://jsk-blog-images.s3.us-west-2.amazonaws.com/books.jpg" /></a>

          <div class="site-info">
            <h1 class="site-name"><a href="/">:)</a></h1>
            <p class="site-description">Jin reviews books</p>
          </div>

          <nav>
            <a href="/">Home</a>
            <a href="/reviews">Index</a>
            <a href="/about">About</a>
          </nav>
        </header>
      </div>
    </div>

    <div id="main" role="main" class="container">
      <article class="page">

  <h1>About</h1>

  <div class="entry">
    <h3 id="what-is-this">What is this?</h3>

<p>Welcome…
A place to where I write my thoughts on books I have read. Due to the volume of books that I read, it is often easy for myself to forget which to recommend to others. For this reason, this page is mainly for myself to reflect on the literature that I come across.</p>

<p>I try to only review books that I have completed - which is the reason the ratings skew towards higher scores. Books not fully completed are marked with a <strong>DNF</strong>. Books I deem too short are marked with a <strong>NR</strong>.</p>

<p>I usually have a personal copy of most reviewed books that I am more than happy to lend out to personal friends.</p>

<h3 id="what-are-you-currently-reading">What are you currently reading?</h3>

<p><em>One Hundred Years of Solitude</em> by Gabriel García Márquez</p>

<h3 id="favorites-books">Favorites books?</h3>

<p>In no particular order:</p>

<p>Nonfiction:</p>
<ul>
  <li><a href="https://jinsung-kim.github.io/The-White-Album/">The White Album</a> - All time California classic.</li>
  <li><a href="https://jinsung-kim.github.io/ametora/">Ametora</a> - Japanese Americana style.</li>
  <li><a href="https://jinsung-kim.github.io/shoe-dog/">Shoe Dog</a> - The origin of Nike.</li>
  <li><a href="https://jinsung-kim.github.io/steve-jobs/">Steve Jobs</a> - A great biography on a fascinating human.</li>
</ul>

<p>Fiction:</p>
<ul>
  <li><a href="https://jinsung-kim.github.io/Brave-New-World/">Brave New World</a> - More relevant than 1984.</li>
  <li><a href="https://jinsung-kim.github.io/The-Stranger/">The Stranger</a> - An entry point to Camus.</li>
  <li><a href="https://jinsung-kim.github.io/Infinite-Jest/">Infinite Jest</a> - Nihilism and free-will woven into a beautiful tale.</li>
  <li><a href="https://jinsung-kim.github.io/Crime-and-Punishment/">Crime and Punishment</a> - A Russian classic.</li>
  <li><a href="https://jinsung-kim.github.io/Calvin-and-Hobbes/">Calvin and Hobbes</a> - Not really “literature”, but a childhood favorite.</li>
</ul>

<h3 id="favorite-authors">Favorite authors?</h3>

<ul>
  <li>David Foster Wallace</li>
  <li>Joan Didion</li>
</ul>

  </div>
</article>

    </div>

    <div class="wrapper-footer">
      <div class="container">
        <footer class="footer">
          
<a href="mailto:jinsungkim1210@gmail.com"><i class="svg-icon email"></i></a>


<a href="https://github.com/jinsung-kim"><i class="svg-icon github"></i></a>
<a href="https://instagram.com/codejin_"><i class="svg-icon instagram"></i></a>







        </footer>
      </div>
    </div>

    

    <script>
    !function(t,e){var o,n,p,r;e.__SV||(window.posthog=e,e._i=[],e.init=function(i,s,a){function g(t,e){var o=e.split(".");2==o.length&&(t=t[o[0]],e=o[1]),t[e]=function(){t.push([e].concat(Array.prototype.slice.call(arguments,0)))}}(p=t.createElement("script")).type="text/javascript",p.crossOrigin="anonymous",p.async=!0,p.src=s.api_host.replace(".i.posthog.com","-assets.i.posthog.com")+"/static/array.js",(r=t.getElementsByTagName("script")[0]).parentNode.insertBefore(p,r);var u=e;for(void 0!==a?u=e[a]=[]:a="posthog",u.people=u.people||[],u.toString=function(t){var e="posthog";return"posthog"!==a&&(e+="."+a),t||(e+=" (stub)"),e},u.people.toString=function(){return u.toString(1)+".people (stub)"},o="init capture register register_once register_for_session unregister unregister_for_session getFeatureFlag getFeatureFlagPayload isFeatureEnabled reloadFeatureFlags updateEarlyAccessFeatureEnrollment getEarlyAccessFeatures on onFeatureFlags onSurveysLoaded onSessionId getSurveys getActiveMatchingSurveys renderSurvey canRenderSurvey canRenderSurveyAsync identify setPersonProperties group resetGroups setPersonPropertiesForFlags resetPersonPropertiesForFlags setGroupPropertiesForFlags resetGroupPropertiesForFlags reset get_distinct_id getGroups get_session_id get_session_replay_url alias set_config startSessionRecording stopSessionRecording sessionRecordingStarted captureException loadToolbar get_property getSessionProperty createPersonProfile opt_in_capturing opt_out_capturing has_opted_in_capturing has_opted_out_capturing clear_opt_in_out_capturing debug getPageViewId captureTraceFeedback captureTraceMetric".split(" "),n=0;n<o.length;n++)g(u,o[n]);e._i.push([i,s,a])},e.__SV=1)}(document,window.posthog||[]);
    posthog.init('ENV["POSTHOG_KEY"]', {
        api_host: 'https://app.posthog.com',
        person_profiles: 'identified_only', // or 'always' to create profiles for anonymous users as well
    })
</script>

  </body>
</html>
