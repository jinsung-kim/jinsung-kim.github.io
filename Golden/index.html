<blockquote>
  <p>The greatest shooter of all time</p>
</blockquote>

<p><img src="https://a.espncdn.com/photo/2016/0510/r82023_1296x729_16-9.jpg" alt="The two time MVP" /></p>

<p>It’s been a while. Since my last post, many things have taken place in which I was unable to take the time to read. After a long hiatus and finding myself with lots of time after graduating, I have returned with the book Golden by Marcus Thompson II. Marcus is a sports columnist that has followed the Golden State Warriors for ten seasons, giving him a lot more insight than others might have, which was the main draw to this book for myself. Given the timing of the Warriors making it to the NBA Finals once more after beating the Dallas Mavericks in 5, and their unrivaled dominance throughout the last decade, I thought it would be a good book to start chipping away at my massive backlog.</p>

<p>Golden is really the first book (at least that I have seen) to provide a deeper look into Wardell Stephen Curry up to the 2017 season. It covers his childhood, his uprising in Charlotte growing up in a Christian household, having an NBA father, to his college days. The book also highlights his rookie year, his MVP seasons and all of the adversaries that he faced throughout, including his many ankle injuries. Marcus does an excellent job pulling sources together to measure the impact of Curry, outside of the three point shooting that he has brought to the league. The detail of his game is written artfully by Marcus, in a way that helped me digest the game at a deeper level than I already have.</p>

<p>I greatly admire Curry, not only as a basketball prodigy but also as a human. He certainly seems to be as great of a role model as the media portrays him to be. His commitment to his family, his grounded nature from his Christian faith and his selflessness on and off the court are attributes that I wish to model my life after. Reading about his work ethic in the off-seasons, his critical nature of his own ability, and his championship mindset made me reflect on myself and suggested ways I could work upon myself.</p>

<p>However in terms of the book, I was disappointed to read that it didn’t really cover anything groundbreaking. Most of what he writes about can be seen as film or accessed through interviews. While I was ok with this, I can imagine anyone who wasn’t trying to relive this time period might find it to be redundant and unnecessary. I am personally not a fan of sport authors writing out each play when a quick Youtube search can visualize the game better than any piece of writing. The game speaks for itself, and often times it felt that Marcus was trying too hard to portray the game a certain way. In addition, each historical series is covered in repeated detail at least three times, often out of chronological order, making it tough to push through certain chapters.</p>

<p>While I greatly enjoyed reading about Curry’s battles with certain superstars, mental fortitude, and his training regiment, this book didn’t quite hit the mark with me. The latter half felt like a chore to push through, and felt that I was finding very occasional gems. I feel grateful to have grown up watching his greatness and wonder where his legacy will fall amongst the Mount Rushmore of NBA greats.</p>

<p>Edit (6/18/22): As I revisit this review, the Golden State Warriors have won another championship and Curry has his first Finals MVP, solidifying his legacy as one of the greatest to ever play the game. To pay homage to his greatness, I have attached a media link below, just click on the image to witness greatness.</p>

<p><img src="https://www.youtube.com/watch?v=G4u82a2lwac" alt="Curry Finals Highlights" />
<a href="https://www.youtube.com/watch?v=G4u82a2lwac"><img src="https://img.youtube.com/vi/G4u82a2lwac/0.jpg" alt="Curry Finals Highlights" /></a></p>

<p><strong>Overall Rating: 5/10</strong></p>
